Gems/Atom/Tools/MaterialEditor/Code/Source/Window/MaterialEditorWindowComponent.cpp

```diff
@@ -18,9 +17,14 @@
 #include <Window/MaterialEditorWindow.h>
 #include <Window/MaterialEditorWindowComponent.h>
 #include <Window/ViewportSettingsInspector/ViewportSettingsInspector.h>
+#include <AtomToolsFramework/Window/AtomToolsMainWindowRequestBus.h>
+
 
 namespace MaterialEditor
 {
+    using FactoryRequestBus = AtomToolsFramework::AtomToolsMainWindowFactoryRequestBus;
+    using RequestBus = AtomToolsFramework::AtomToolsMainWindowRequestBus;
```

Comment:
These abbreviations are too generic.  Coding standard does not allow them outside of functions anyway.  It's OK to use the full names.  

Those behavior context reflection spec should probably be moved to ATF in a future PR.

We'll also have to update all the material editor specific strings to use the new atom tools based function and class names in a future PR where we will also have to update the test scripts.